src/kimmdy/runmanager.py

```diff
@@ -400,6 +391,139 @@ def _setup(self, files: TaskFiles) -> TaskFiles:
         logger.info("Done with setup")
         return files
 
+    def _restart_from_rundir(self):
+        """Set up RunManager to restart from a run directory"""
+
+        task_dirs = get_step_directories(self.config.restart.run_directory, "all")
```

Comment:
I guess now is as good a time as any to get consistent with calling things tasks or steps.
I belive things in the sequence are meant to be tasks, while things in the recipe are steps.

Reply:
done, at least for this case. Do you know of any other cases where we confused those?